test/markets/markets.e2e-spec.ts

```diff
@@ -0,0 +1,66 @@
+import { INestApplication, ValidationPipe } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { MainModule } from '../../src/main.module';
+
+const request = require('supertest');
+
+describe('Markets', () => {
+
+  let app: INestApplication;
+
+  beforeAll(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [MainModule],
+    }).compile();
+
+    app = moduleFixture.createNestApplication();
+
+    // Validation pipeline is require to check validations.
+    app.useGlobalPipes(
```

Comment:
I'm getting the impression that this is something we can re-use for all tests. In which case, can we create a separate file that has the boilerplate initialization for these tests? and then we import it where we need it

Reply:
Moved app initialization to separate file. It is still not reusing main initialisation, couldn't get it working. NestJs bootstrap is slightly different from DTFS API bootstrap.